content/source/docs/cloud/registry/using.html.md

```diff
@@ -17,16 +17,17 @@ A Terraform Cloud private module registry has a few key differences from the [pu
 - **Authentication:** Terraform Cloud workspaces using version 0.11 and higher can automatically access your private modules during Terraform runs. But when you run Terraform on the command line, you must [authenticate](/docs/cloud/registry/using.html#authentication) to Terraform Cloud or your Terraform enterprise instance.
 
 
-## Finding Private Modules
+## Finding Modules
 
-All users in your organization can view your private module registry. To find available modules, click the "Modules" button in the Terraform Cloud main navigation bar.
+All users in your organization can view your private module registry. To find available modules, click the "Registry" button in the Terraform Cloud main navigation bar.
 
-![Terraform Cloud screenshot: Navigation bar with modules button highlighted](./images/using-modules-button.png)
+![Terraform Cloud screenshot: Navigation bar with registry button highlighted](./images/using-modules-button.png)
```

Comment:
This image also still has "Modules" in the menu bar. Should it?

Reply:
I did not update all images.  Only the ones I could.  I don't have TFE server to update images related to shared organizations.  Originally, I was only going to update index page and add my new add public modules page, but then decided to improve other pages too.  I figured making corrections was better than leaving them the way they were even without updating images.  Perhaps you can update the other images?